dali/operators/image/remap/remap.cu

```diff
@@ -61,6 +64,27 @@ class RemapGpu : public Remap<GPUBackend> {
                {}, {}, make_span(interps_), {});
   }
 
+
+  /**
+   * Stores and returns a NppRemapKernel object. Creates one if it doesn't exist.
+   */
+  template<typename InputType>
+  kernels::remap::NppRemapKernel<StorageGPU, InputType> &GetOrCreateKernel(int device_id) {
+    assert(device_id >= 0);
+    static std::unordered_map<int /* device_id */,
+            kernels::remap::NppRemapKernel<StorageGPU, InputType>> kernel_map;
```

Comment:
this doesn't need to be static. It could just be an operator member (cache per op instance). Without knowing too much of the nature of NppRemapKernel, I think it'd be a better practice.
Just a question: Does NppRemapKernel keep any state? Can it be used from different threads?

Reply:
KernelManager field in the operator is meant for that, right?

Reply:
Went for KernelManager